bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableHBaseVersion.java

```diff
@@ -30,18 +30,19 @@
   @Test
   public void testVersionNumber() {
     // Version must have 3 parts i.e. x.y.z and can have an optional of -SNAPSHOT suffixed.
-    Pattern versionPattern = Pattern.compile("^(\\d+)\\.(\\d+)\\.(\\d+)(-\\w+)?");
+    Pattern versionPattern = Pattern.compile("^(\\d+)\\.(\\d+)\\.(\\d+)([-\\w]+)?");
     Matcher versionMatcher = versionPattern.matcher(BigtableHBaseVersion.getVersion());
 
     assertTrue("incorrect version format", versionMatcher.matches());
 
     int result =
         ComparisonChain.start()
-            .compare(1, Integer.parseInt(versionMatcher.group(1)))
-            .compare(14, Integer.parseInt(versionMatcher.group(2)))
-            .compare(1, Integer.parseInt(versionMatcher.group(3)))
+            .compare(2, Integer.parseInt(versionMatcher.group(1)))
+            .compare(0, Integer.parseInt(versionMatcher.group(2)))
+            .compare(0, Integer.parseInt(versionMatcher.group(3)))
```

Comment:
let's add in a small assertion for the `-alpha`

Reply:
I dont want to update this test for every release. It's just meant as a sanity test that the current version we are releasing is > 2.0.0

Reply:
I updated the test to pattern match the suffix as well